kafka-client/src/main/java/dev/responsive/kafka/internal/db/LwtWriter.java

```diff
@@ -34,22 +37,38 @@ public class LwtWriter<K> implements RemoteWriter<K> {
   private final Supplier<BatchableStatement<?>> fencingStatementFactory;
   private final RemoteTable<K, BoundStatement> table;
   private final int partition;
-  private final int batchSize;
+  private final long maxBatchSize;
 
   private final List<BatchableStatement<?>> statements;
 
   public LwtWriter(
+      final CassandraClient client,
+      final Supplier<BatchableStatement<?>> fencingStatementFactory,
+      final RemoteTable<K, BoundStatement> table,
+      final int partition
+  ) {
+    this(
+        client,
+        fencingStatementFactory,
+        table,
+        partition,
+        MAX_BATCH_SIZE
```

Comment:
This is the only real/logical change: we now always just pass in the actual `MAX_BATCH_SIZE` (although we still parameterize it within this class rather than just using hard-coding the constant directly so we can set it to something lower for the unit tests)